functions/src/export-csv.ts

```diff
@@ -293,8 +292,14 @@ function getMultipleChoiceValues(
     responses.selectedOptionIds?.map(
       id => getMultipleChoiceLabel(task, id) || '#ERR'
     ) || [];
-  if (responses.otherText && responses.otherText.trim() !== '')
-    values.push(responses.otherText);
+  // Ensure at least one value is present: if no values are selected and 'otherText' is empty, add 'Other' as a fallback.
```

Comment:
Please add a note that this is a temporary workaround for an issue on the Android client and add a link to the bug.

Reply:
@rfontanarosa Reminder to add a note here.